README.md

```diff
@@ -20,19 +20,30 @@ The following instructions assume you are using Python 2 and a Debian-based (lik
 
 ## Dependencies
 
-Pymavlink requires 3 dependencies :
+Pymavlink requires 4 dependencies :
 
     - future : for python 2 and python 3 interoperability (http://python-future.org/)
     - lxml : for checking and parsing xml file (http://lxml.de/installation.html)
     - python-dev : for mavnative
+    - gcc : for mavnative
```

Comment:
Will any C compiler work?

Reply:
no, both pip and setup.py will try to call gcc by default

Reply:
at least on linux